sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcRelation.scala

```diff
@@ -253,7 +260,7 @@ private[orc] case class OrcTableScan(
     maybeStructOI.map { soi =>
       val (fieldRefs, fieldOrdinals) = nonPartitionKeyAttrs.map {
         case (attr, ordinal) =>
-          soi.getStructFieldRef(attr.name.toLowerCase) -> ordinal
```

Comment:
From the unit test, seems this is the only place we need to update. 
However in Hive, all of the schema is case insensitive.
e.g.
```
create table a(AAA int, bb string);
desc a;
outputs:
aaa   int
bb     string
```
So I think we need to change this, is that correct?